inscopix/pyisx
# IT-1228: Refactor repo to use isxcore C++ API

## What type of PR is this? (check all applicable)


## Description
Refactor the python API so that it's no longer a pure python implementation. Instead this API is a wrapper around the `isxcore` C++ API, used in the IDPS GUI and IDPS Python/MATLAB APIs.

## Related Issues & Documents

- Related Issue #
- Closes #

## Testing Instructions

Added all python unit tests from IDPS repo.

* Follow the setup instructions in the [README](https://github.com/inscopix/pyisx/blob/34191df5a45cae1fc01fc3230f7730ef1105ccb6/README.md)
* Ensure you have the third party and test data folders used to build/test IDPS available on your machine (will be in mosaic2 dir)
* Build the lib and run the unit tests: make test THIRD_PARTY_DIR=/path/to/mosaic2/third_party TEST_DATA_DIR=/path/to/mosaic2/test_data


## Added/updated tests?


      have not been included_
